pkg/util/oc/pod_commands.go

```diff
@@ -83,3 +90,14 @@ func WaitAllPodsReady(t test.TestHelper, ns string) {
 	t.T().Helper()
 	shell.Executef(t, `oc -n %s wait --for condition=Ready --all pods --timeout 180s`, ns)
 }
+
+func WaitCondition(t test.TestHelper, ns string, kind string, name string, condition string) {
+	t.T().Helper()
+	retry.UntilSuccess(t, func(t test.TestHelper) {
```

Comment:
```suggestion
	retry.UntilSuccessWithOptions(t, retry.Options().MaxAttempts(30), func(t test.TestHelper) {
```

Reply:
The reason why we should specify maxAttempts=30 here is because we're waiting for 10s in each attempt. And because we may later change the default in `UntilSuccess` and we don't want that change to affect this function here.